src/main/java/gregtech/integration/jei/recipe/primitive/BasicRecipeCategory.java

```diff
@@ -14,6 +14,7 @@
 import java.util.Collections;
 import java.util.List;
 
+//TODO, move this and all inheritors into integration.jei.basic
```

Comment:
Um done, right?

Reply:
No. I was holding off on doing it in this PR because originally someone was working on JEI stuff at the same time, but it seems fine to do it now.